readme.md

```diff
@@ -1,4 +1,7 @@
+![Polaris Build](https://github.com/adamshapiro0/polaris/workflows/Polaris%20Build/badge.svg?branch=master)
```

Comment:
odd this URL is showing up under you instead of Point One?

Reply:
Oh whoops, I prototyped this on a fork of the repo so I could test release tagging/generation without creating a bunch of releases on the public repo.